src/Dhall/Pretty/Internal.hs

```diff
@@ -624,29 +709,29 @@ prettyKeyValue separator (key, value) =
   where
     long = Pretty.hardline <> "    " <> prettyExprA value
 
-prettyRecord :: Pretty a => InsOrdHashMap Text (Expr s a) -> Doc ann
+prettyRecord :: Pretty a => InsOrdHashMap Text (Expr s a) -> Doc Ann
 prettyRecord =
-    braces . map (prettyKeyValue ":") . Data.HashMap.Strict.InsOrd.toList
+    braces . map (prettyKeyValue colon) . Data.HashMap.Strict.InsOrd.toList
 
-prettyRecordLit :: Pretty a => InsOrdHashMap Text (Expr s a) -> Doc ann
+prettyRecordLit :: Pretty a => InsOrdHashMap Text (Expr s a) -> Doc Ann
 prettyRecordLit a
     | Data.HashMap.Strict.InsOrd.null a =
-        "{=}"
+        literal "{=}"
```

Comment:
I would classify this as `Syntax`, or at least the outer braces should be treated syntax for consistency with non-empty records

Reply:
Yea, I couldn't really decide on this. I think you could argue it either way. I might annotate the whole thing as a literal, but then the braces as syntax.